hypha/apply/funds/templates/funds/includes/dialog_form_base.html

```diff
@@ -0,0 +1,53 @@
+{% load i18n %}
+<form
+    class="form {{ extra_classes }}"
+    method="post"
+    hx-post="{{ url }}"
+    hx-target="{{ target }}"
+    hx-swap="innerHTML"
+    hx-select="{{ target }}"
+    id="{% if form_id %}{{ form_id }}{% else %}{{ form.name }}{% endif %}"
+    enctype="multipart/form-data"
+    {% if action %}action="{{ action }}"{% endif %}>
+<!--    hx-on::after-request="if(event.detail.successful){leadForm = false, console.log(leadForm)}",>-->
```

Comment:
@theskumar Have you used after-request event? Although it is working fine and even leadForm is set as False but dialog box is not closing. If it can work and close the dialog box on setting leadForm false then we can remove the '@click' trigger from submit button. It'll help in error cases.